release/downloadIstioCandidate.sh

```diff
@@ -21,13 +21,13 @@
 # The script fetches the latest Istio release candidate and untars it.
 # You can pass variables on the command line to download a specific version
 # or to override the processor architecture. For example, to download
-# Istio 1.6.8 for the x86_64 architecture,
-# run curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.6.8 TARGET_ARCH=x86_64 sh -.
+# Istio 1.6.8 for the x86_64 architecture and linux OS,
+# run curl -L https://istio.io/downloadIstio | ISTIO_VERSION=1.6.8 TARGET_ARCH=x86_64 TARGET_OS=Linux sh -.
 
 set -e
 
 # Determines the operating system.
-OS="$(uname)"
+OS="{TARGET_OS:-$(uname)}"
```

Comment:
Giving you a way to test before you merge, but here is what is likely the fix.
```suggestion
OS="${TARGET_OS:-$(uname)}"
```

Reply:
Actually if I specify `TARGET_OS=Linux` (I had specified a lower case l in my earlier testing), it seems to work.

Reply:
Looking we don't seem to do any validation of the TARGET_ARCH either.

Reply:
Yes, this script does not strictly handle the values of OS/ARCH. The reason I used "Linux" here is to make the following "ARCH_SUPPORTED" section of code work.

Reply:
name does return `Linux` on a Ubuntu VM and expected on others.